packages/models-library/src/models_library/settings/docker_registry.py

```diff
@@ -0,0 +1,26 @@
+from typing import Optional
+
+from pydantic import Field, SecretStr
+
+from .base import BaseCustomSettings
+
+
+class RegistrySettings(BaseCustomSettings):
+
+    REGISTRY_AUTH: bool = Field(..., description="do registry authentication")
+    REGISTRY_PATH: Optional[str] = Field(
+        None, description="development mode only, in case a local registry is used"
+    )
+    REGISTRY_URL: str = Field("", description="url to the docker registry")
```

Comment:
AnyUrl vs str?

Reply:
unfortunately no, the naming is misleading will add a comment